sdk/azidentity/azidentity.go

```diff
@@ -16,6 +18,31 @@ const (
 	defaultAuthorityHost = "https://login.microsoftonline.com/"
 )
 
+// log environment variables that can be used for credential types
+func logEnvVars() {
+	// Log available environment variables
+	envVars := []string{}
+	log := azcore.Log()
+	if envCheck := os.Getenv("AZURE_TENANT_ID"); len(envCheck) > 0 {
+		envVars = append(envVars, "AZURE_TENANT_ID")
+	}
+	if envCheck := os.Getenv("AZURE_CLIENT_ID"); len(envCheck) > 0 {
+		envVars = append(envVars, "AZURE_CLIENT_ID")
+	}
+	if envCheck := os.Getenv("AZURE_CLIENT_SECRET"); len(envCheck) > 0 {
+		envVars = append(envVars, "AZURE_CLIENT_SECRET")
```

Comment:
Is AZURE_CLIENT_SECRET considered PII?

Reply:
Good point, we should replace the actual value with `<REDACTED>`.

Reply:
I'm not printing the value of the environment variable. I'm literally just printing the name of the variable for the user's knowledge so that they know this variable is set in the environment. If you look at line 42 you can see that I'm just printing the names of the variables here for the user to know what the environment is like that they're operation in. Also this array of env vars is not used anywhere else so it won't run the risk of adding client secret anywhere else.

Reply:
Got it! Thanks for the clarification.